ui/tests/acceptance/clients/counts/overview-test.js

```diff
@@ -235,102 +235,59 @@ module('Acceptance | clients | overview', function (hooks) {
   });
 });
 
-module('Acceptance | clients | overview | sync in license, activated', function (hooks) {
+module('Acceptance | clients | overview | secrets sync', function (hooks) {
```

Comment:
i found all the separate modules of this test really hard to parse, so combined them into one so `setupApplicationTest` and `setupMirage` only need to be called once.